samples/SortBenchmark/DiskHelper.fs

```diff
@@ -41,13 +41,12 @@ open Prajna.Tools
 open Prajna.Tools.FSharp
 open Prajna.Tools.StringTools
 open Prajna.Core
-//open Prajna.Service.CoreServices
 
 [<Serializable>]
 type StorageProfile (profileName : string) =
     //member val diskProfile = [|"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"c:\\sortbenchmark\\";"d:\\sortbenchmark\\";"e:\\sortbenchmark\\";"f:\\sortbenchmark\\"|] with get
-    member val drives = DriveInfo.GetDrives()
-    member val diskProfile = DriveInfo.GetDrives() |> Array.filter(fun di -> di.DriveType = DriveType.Fixed)  |> Array.filter(fun di -> di.TotalSize > 100000000000L) |> Array.map(fun di -> Path.Combine(di.Name,"sortbenchmark").ToLower()+"\\")
+    //member val drives = DriveInfo.GetDrives()
+    member val diskProfile = null with get,set
```

Comment:
Can it be initialized with a Lazy<'a[]>? If this is meant to be accessed only at the remote machine, a Lazy has another advantage: it let's you throw an informative exception in case one tries to access it at the client. Could be as simple as 'failwith "Shouldn't be accessing this at the client"', which is way better than a NullReferenceException, and making everyone check for null everywhere.
To be clear: I only propose implementing this before the pull if it is trivial and doesn't add any delay to the commit. Otherwise, we should pull this as is and change (or not) later. My general take on code reviews is that, as long as the code is useful and not harmful outside its context (and this clearly qualifies), commited code is always better than code sitting in your repo.